Abstract (#78)
MrLongbottom/P10-Article: sections/Abstract.tex

We introduce two topic models that extend \gls{lda} to include author and category metadata information and a third model which applies taxonomy metadata on the \gls{pam}.
The author-topic and category-topic models are based on the author-topic model from \citet{author_topic_2012} with slight modifications, and the taxonomy-topic model uses the \gls{pam} from \citet{li2006pachinko}.
To make the \gls{pam} include the metadata information in each layer of its \gls{dag} structure, a novel topic locking mechanism is created.

> The results show that while, for the applied dataset, the author-topic and category-topic models get a lower topic coherence than \gls{lda}, the taxonomy-topic model has the highest topic coherence and most understandable topics.

Reviewer: ```suggestion
The results show that while, for the applied dataset, the author-topic and category-topic models get a lower topic coherence than \gls{lda}.  
The taxonomy-topic model has the highest topic coherence and most understandable topics.
```

Reviewer 2: hvis det skal være to sætninger kan vi ikke bruge 'while' da det bliver brugt til at sammenligne to ting.

Reviewer 3: ```suggestion
The results show that while, for the applied dataset, the author-topic and category-topic models get a lower topic coherence than \gls{lda}.  The taxonomy-topic model has the highest topic coherence and most understandable topics.
```

Reviewer 2: ```suggestion
The results show that for the applied dataset, the author-topic and category-topic models get a lower topic coherence than \gls{lda}.
However, the taxonomy-topic model has a higher topic coherence and more understandable topics than \gls{lda}.
```
